plugins/tiddlywiki/multiwikiclient/configOfficialPluginLibrary.tid

```diff
@@ -0,0 +1,7 @@
+title: $:/config/OfficialPluginLibrary
+tags: $:/tags/PluginLibrary
+url: https://tiddlywiki.com/library/v5.1.23/index.html
```

Comment:
v5.1.23 ?? Is this version number intended?

Reply:
Ouch – that's actually an issue with the core;  the same file is at https://github.com/Jermolene/TiddlyWiki5/blob/master/plugins/tiddlywiki/tiddlyweb/configOfficialPluginLibrary.tid